### ISSUE TITLE
WCAG 1.4.3: Contrast (Minimum)

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#5116]
## WCAG 1.4.3: Contrast (Minimum)

**Level:** AA
**Principle:** Perceivable
**Result:** *pending*

### Description
This issue tracks the audit and implementation of WCAG 2.2 Success Criterion 1.4.3: Contrast (Minimum).

### Resources
[WCAG 2.2 Success Criterion 1.4.3](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html)

> The visual presentation of [text](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html#dfn-text) and [images of text](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html#dfn-image-of-text) has a [contrast ratio](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html#dfn-contrast-ratio) of at least 4.5:1, except for the following:

> Large Text
[Large-scale](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html#dfn-large-scale) text and images of large-scale text have a contrast ratio of at least 3:1;

> Incidental
Text or images of text that are part of an inactive [user interface component](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html#dfn-user-interface-component), that are [pure decoration](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html#dfn-pure-decoration), that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.

> Logotypes
Text that is part of a logo or brand name has no contrast requirement.

### Audit Checklist
- [ ] Review previous Audit Result
- [ ] Review work identified in previous audit and list open issues.
- [ ] Check for any other relevant open issues.
- [ ] Retest open issues, are these issues still present?
- [ ] Test for any new issues that may have been introduced.
- [ ] Determine compliance status, add this to issue name.
- [ ] Add any new issues to the Accessibility project for triage.

### Previous audit
#4127
> ✅ contrast
Janeway is designed so that each journal can have their own colour schemes making colour 'user-content'. We are exploring how we can support good colour contrast choices.

 ### Progress since last audit

 ### Audit Notes

 ### Work arising
